infer/src/integration/Maven.ml

```diff
@@ -10,6 +10,7 @@ open! IStd
 
 module CLOpt = CommandLineOption
 module L = Logging
+module F = Filename
```

Comment:
The (completely undocumented) style in infer is to abbreviate some modules we commonly use, but `Filename` is not one of them, and `F` stands for `Format` elsewhere in the code. I'll try and document this.

tl; dr: please remove this line and use the full module name below instead of `F`.